internal/publish/model/stats.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package model is a model abstraction for health authority telemetry.
+package model
+
+import (
+	"time"
+)
+
+const (
+	MaxOldestTEK = 14
```

Comment:
Should these max ages be config?

Reply:
I don’t think so in this case. oldtest TEK age is bounded by what devices will report - although, I think we should probably make that 15

and the max onset days is what we now to be the largest epi relivent date (And that’s a bit of a stretch.

this is also the collection - display will be in verification server.

Reply:
Yea verification takes 14 but 15 if tzOffset is negative (just to be careful about time zones)

Reply:
ok - I changed this so that it is 16
0-15 and > 15

and 0-18 and > 28